tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -171,29 +171,54 @@ export function getReleaseTag(apiItem: ApiItem): ReleaseTag | undefined {
 }
 
 /**
- * Creates a string representation of the provided release tag.
+ * Gets all {@link https://tsdoc.org/pages/spec/tag_kinds/#modifier-tags | modifier tags} associated with the provided API item.
  *
- * @remarks If `None`, this will return an empty string.
+ * @remarks Note that this will include both standard and any preserved custom tags.
+ *
+ * @public
  */
-export function releaseTagToString(releaseTag: ReleaseTag): string {
```

Comment:
Was unused, so I removed it.

Reply:
(I had this removal locally when I started looking into this project. ;) )